src/domain/NewNotificationRepository.ts

```diff
@@ -61,6 +97,41 @@ export const makeNewNotificationResponse =
     notificationRequestId,
   });
 
+export const matchNewNotificationRecordCriteria = pipe(
+  existsApiKey,
+  P.and(hasRecipientTaxId),
+  P.and(hasRecipientDigitalDomicile),
+  P.and(hasPhysicalAddress),
+  P.and(hasRegisteredLetterAsPhysicalDocumentType),
+  P.and(hasRecipientPaymentCreditorTaxId),
+  P.and(hasRecipientPaymentNoticeCode),
+  P.and(hasSuccessfulResponse)
+);
```

Comment:
Remove it if you choose to move them into the [sendPaymentNotificationChecklist.ts](https://github.com/pagopa/pn-local-emulator/pull/67/files#diff-a94b02017c200eab4581cf751d371018a465dd8eecd987451d69de94d99e5a2c) file

Reply:
Done!